src/js/cure-center-content.js

```diff
@@ -124,12 +182,26 @@ function getListIcon(type) {
     return icon[type] || icon['default'];
 }
 
-setTimeout(()=>updatePage(), 1000);
+function observeEntryDisplayResize() {
+    const entryDisplay = $('#entry_display')[0];
+    const observer = new ResizeObserver(entries => {
+        entries.forEach(entry => {
+            setTimeout(() => $('#entry-column').outerWidth(getEntryColumnWidth()), 300);
```

Comment:
I ended up having to put them in because the labarchives pages load pretty slowly.  I tried to find other methods than timeout, but nothing seemed to work. Injecting the code at start, idle, end didn't matter.  Tried various window.onload, document.ready, etc... without success.  If there's another way, I would rather do that.

Reply:
Thanks Ben!  I changed up the code to remove all the arbitrary setTimeouts, added a defer function and made more use of mutation observers.  let me know what you think.  Thanks!